utils/customize_node.py

```diff
@@ -188,9 +201,11 @@ def set_apt_proxy(args):
     # insufficient.  Either way, get the proxies from os.environ.
     path = args.new_fs_dir + '/etc/apt/apt.conf.d'
     file_utils.make_dir(path)
-    path += '/00FAMproxy.conf'  # !!! FIXME: no hardcoded custom file names !!!
+    path += '/00TMMS.conf'
```

Comment:
Renamed to a bit more generic name to indicate that this config was generated by "tm-manifesting'.